packages/smooth_app/lib/cards/product_cards/product_title_card.dart

```diff
@@ -64,14 +54,41 @@ class ProductTitleCard extends StatelessWidget {
         ),
       ];
     } else {
+      final Size imageSize =
+          Size.square(MediaQuery.sizeOf(context).width * 0.25);
+
       children = <Widget>[
         Padding(
           padding: const EdgeInsetsDirectional.only(top: SMALL_SPACE),
           child: IntrinsicHeight(
             child: Row(
               crossAxisAlignment: CrossAxisAlignment.start,
               children: <Widget>[
-                const _ProductTitleCardPicture(),
+                TooltipTheme(
+                  data: TooltipThemeData(
+                    verticalOffset: imageSize.width / 2,
+                    preferBelow: true,
+                  ),
+                  child: ProductPicture(
+                    product: product,
+                    imageField: ImageField.FRONT,
+                    fallbackUrl: product.imageFrontUrl,
+                    size: imageSize,
+                    showObsoleteIcon: true,
+                    imageFoundBorder: 1.0,
+                    imageNotFoundBorder: 1.0,
+                    borderRadius: BorderRadius.circular(14.0),
+                    onTap: () async => Navigator.push<void>(
+                      context,
+                      MaterialPageRoute<bool>(
```

Comment:
void or bool?

Reply:
Actually, that's a copy/paste from the current code.
I'll fix that